Mathlib/CategoryTheory/Adjunction/FullyFaithful.lean

```diff
@@ -190,6 +194,15 @@ lemma isIso_unit_app_iff_mem_essImage [R.Faithful] [R.Full] {Y : C} :
     rw [NatTrans.isIso_app_iff_of_iso _ i.symm]
     infer_instance
 
+/-- If `η_A` is an isomorphism, then `A` is in the essential image of `i`. -/
+theorem mem_essImage_of_unit_isIso [R.Faithful] [R.Full] (A : C)
+    [IsIso (h.unit.app A)] : A ∈ R.essImage :=
+  ⟨L.obj A, ⟨(asIso (h.unit.app A)).symm⟩⟩
+#align category_theory.mem_ess_image_of_unit_is_iso CategoryTheory.Adjunction.mem_essImage_of_unit_isIso
+
+@[deprecated (since := "2024.06.19")] alias _root_.CategoryTheory.mem_essImage_of_unit_isIso :=
+  mem_essImage_of_unit_isIso
+
```

Comment:
I hope this is the correct syntax for depracation. Do you agree that this lemma belongs in the `Adjunction` namespace?

Reply:
The date should be `2024-06-19` instead of `2024.06.19`.